clients/admin-ui/src/features/common/table/v2/cells.tsx

```diff
@@ -203,7 +203,7 @@ export const DefaultHeaderCell = <T,>({
   </Text>
 );
 
-interface EnableCellProps extends Omit<SwitchProps, "value"> {
+interface EnableCellProps extends Omit<SwitchProps, "value" | "onToggle"> {
```

Comment:
updated `type` definition means these `onToggle` types are no longer compatible so we needed to omit here in favor of our own definition for this use case.